services/dbt/target/run/data_pipelines/models/datenspende/daily_vital_features/aggregates_of_vitals_before_infection.sql

```diff
@@ -0,0 +1,51 @@
+
+
+  create  table "rocs"."jakob"."aggregates_of_vitals_before_infection__dbt_tmp"
```

Comment:
I think we should stick to creating tables tables in the common schemas (e.g. datenspende_derivatives in this case) outside of test/develop so people know where to find stuff

Reply:
This is resolved with the comment below.